feconf.py

```diff
@@ -549,6 +549,7 @@ def get_empty_ratings():
 FRACTIONS_LANDING_PAGE_URL = '/fractions'
 LEARNER_DASHBOARD_URL = '/learner_dashboard'
 LEARNER_DASHBOARD_DATA_URL = '/learnerdashboardhandler/data'
+LEARNER_DASHBOARD_FEEDBACK_THREAD_URL = '/learnerdashboardthread'
```

Comment:
This is a json handler, right? By convention it should end with DATA_URL and the url should end with 'handler'.